resize-observer/devicepixel2.html

```diff
@@ -0,0 +1,152 @@
+<!doctype html>
+<link rel="match" href="devicepixel2-ref.html">
+<meta name="assert" content="Device pixel content box sizes and pixel snapping are correct in Resize Observer callback">
+<style>
+  html, body {
+    margin: 0;
+  }
+  .outer {
+    width: 100px;
+    display: flex;
+  }
+  .outer>* {
+    flex: 1;
+    height: 20px;
+  }
+</style>
+<body>
+  <div></div>
+</body>
+<script>
```

Comment:
nit: let's move the closing </body> to the end (as you did for the reference case), to reflect the reality of the actual DOM structure here.